build.rs

```diff
@@ -370,46 +369,52 @@ fn make_elfutils(compiler: &cc::Tool, src_dir: &path::Path, _: &path::Path) {
     emit_rerun_directives_for_contents(&src_dir.join("elfutils").join("src"));
 }
 
-fn make_libbpf(
-    compiler: &cc::Tool,
-    cflags: &ffi::OsStr,
-    src_dir: &path::Path,
-    out_dir: &path::Path,
-) {
-    let src_dir = src_dir.join("libbpf/src");
-    // create obj_dir if it doesn't exist
-    let obj_dir = path::PathBuf::from(&out_dir.join("obj").into_os_string());
-    let _ = fs::create_dir(&obj_dir);
-
-    let status = process::Command::new("make")
-        .arg("install")
-        .arg("-j")
-        .arg(&format!("{}", num_cpus()))
-        .env("BUILD_STATIC_ONLY", "y")
-        .env("PREFIX", "/")
-        .env("LIBDIR", "")
-        .env("OBJDIR", &obj_dir)
-        .env("DESTDIR", out_dir)
```

Comment:
Are these all irrelevant or how are they picked up now?

Reply:
> Are these all irrelevant or how are they picked up now?

Now, all the generated files will be in the project directory. I think these environment variables are not important, so I deleted them.

Reply:
Well, `BUILD_STATIC_ONLY` sounds as if it should be set if we are building a static library, for one reason or another. Just removing it because you think it may not important isn't really going to cut it from my perspective, we are going to need a bit more than that.